lib/components/views/GotVeggies.js

```diff
@@ -0,0 +1,96 @@
+import React, {Component, PropTypes} from 'react';
+import {bindActionCreators} from 'redux';
+import {connect} from 'react-redux';
+import {FontIcon, RaisedButton} from 'material-ui';
+import DonationSizeChooser from 'components/DonationSizeChooser';
+
+import * as userActions from 'actions/userActions';
+import {getQueryVariable} from '../../utils/urlUtils';
+
+@connect(
+  state => ({user: state.auth.user}),
+  dispatch => bindActionCreators(userActions, dispatch)
+)
+export default class GotVeggies extends Component {
+  static propTypes = {
+    user: PropTypes.object,
+    setDonationPreference: PropTypes.func
+  };
+
+  static childContextTypes = {
+    muiTheme: React.PropTypes.object
+  };
+
+  constructor(props) {
+    super(props);
+    // donation sizes
+    // 0 - 0 donations
```

Comment:
In the future it would be nice to have these values in a map that any time we are dealing with them we reference them by key.